test/system/post_test.rb

```diff
@@ -124,27 +124,30 @@ def assert_page_does_not_reload(message = "page should not reload")
     wiki = nodes(:wiki_page)
 
     visit wiki.path
+
     # save text of wiki before edit
-    old_wiki_content = find("#content").text
+    old_wiki_content = find("#content p").text
 
     find("a#edit-btn").click()
     find("#text-input").set("wiki text")
     find("a#publish").click()
 
     # view wiki
-    current_wiki_content = find("#content").text
+    current_wiki_content = find("#content p").text
```

Comment:
Sorry this might pass in testing but it doesn't accomplish the required purpose. The `#content` div is composed of different elements like headings, paragraphs and many others. Not just p tags

Reply:
Not really, there is only one p tag, I can add :first-of-type if you insist though.

Reply:
Still better than `#content` alone.

Reply:
For example, if you go to https://publiclab.org/about/. You will see the #content div has many different elements. Is there a way to get the text of all those elements and not just the p tags?

Reply:
This is testing page.

Reply:
Alright; but say later someone edits the test and adds a heading. The test will fail because the p tags will not have changed so the content will be marked equal

Reply:
No, it won't fail, the `p` tag is first and it would fetch it.

Reply:
It failed again :(

Reply:
I think it's because of `visit`, let's try again by setting fixed path.